tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -21,6 +21,7 @@
 import functools
 from tensorflow_recommenders_addons.dynamic_embedding.python.ops.dynamic_embedding_creator import KVCreator
 import six
+import horovod.tensorflow as hvd
```

Comment:
Please format all the files firstly, refer to: https://github.com/tensorflow/recommenders-addons/blob/master/STYLE_GUIDE.md

Reply:
done.